frontend/src/util/window.ts

```diff
@@ -1,42 +1,16 @@
-import { Admin } from '@graph/schemas'
-import { INTERCOM_APP_ID } from '@util/constants/constants'
-import { H } from 'highlight.run'
+import { message } from 'antd'
 
 import { FRONTEND_URI } from '@/constants'
 
 export function GetBaseURL(): string {
 	return FRONTEND_URI
 }
 
-interface IntercomInit {
-	admin?: Admin
+export const showSupportBubble = () => {
+	window.open('https://highlight.io/community', '_blank')
 }
 
-export function loadIntercom({ admin }: IntercomInit = {}) {
-	const config = {
-		app_id: INTERCOM_APP_ID,
-		alignment: 'right',
-		hide_default_launcher: true,
-		email: admin?.email,
-		user_id: admin?.uid,
-	}
-
-	H.getSessionURL()
-		.then((sessionUrl) => {
-			window.Intercom('boot', {
-				...config,
-				sessionUrl,
-			})
-		})
-		.catch(() => {
-			window.Intercom('boot', config)
-		})
-}
-
-export const showIntercomBubble = () => {
-	window.Intercom('update', { hide_default_launcher: false })
-}
-
-export const showIntercomMessage = (message = '') => {
-	window.Intercom('showNewMessage', message)
+export const showSupportMessage = async (_ = '') => {
```

Comment:
Looks like we aren't using the message that is provided in the method call (e.g. `'Hi! I would like to use the session export feature.'`), and wanted to make sure that this was intended

Reply:
yeah, wasn't sure what to best do with it; going to add it to the popup